frontend/app_flowy/lib/workspace/presentation/stack_page/doc/styles.dart

```diff
@@ -83,9 +84,11 @@ DefaultStyles customStyles(BuildContext context) {
           TextStyle(color: baseStyle.color!.withOpacity(0.6)),
           baseSpacing,
           const Tuple2(6, 2),
-          BoxDecoration(
+          const BoxDecoration(
+            //color needs a constant
+            // color: (theme.isDark ? theme.shader1 : theme.shader7), //Fix color for block qoute dark theme
             border: Border(
-              left: BorderSide(width: 4, color: Colors.grey.shade300),
+              left: BorderSide(width: 4, color: Colors.blueAccent),
```

Comment:
Try not to use a specific color, use a placeholder. This won't work when we add a 3rd theme (or even better make the app themable)